rootdir/vendor/etc/init/devstart.rc

```diff
@@ -4,3 +4,11 @@ service devstart_sh /vendor/bin/init.qcom.devstart.sh
     group root system
     disabled
     oneshot
+
+# ADSP devices
+on property:vendor.qcom.adspup=1
```

Comment:
Shouldn't this be condensed into `on property:vendor.qcom.adspup=1 && property:vendor.qcom.cdspup=1`? That way both devices are guaranteed to be started before starting SLPI.

On devices that don't have `cdsp` this doesn't matter, because their script will (even when failing to `echo 1` to the file) set `cdspup` to `1`.

Reply:
The `init.cdspstart.sh` and `cdsprcpd.rc` files only get copied for sdm845 devices
See https://github.com/sonyxperiadev/device-sony-common/blob/master/rootdir/Android.mk#L262-L279

Reply:
Disregard the last sentence. This'll be a race condition only on sdm845 then.

Reply:
Consensus: Let's not worry about this. Tama is already fine with `SLPI` started _before_ adsp/cdsp, so it shouldn't matter if it's started inbetween those.